crypto/evp/evp_lib.c

```diff
@@ -524,7 +534,7 @@ unsigned long EVP_MD_flags(const EVP_MD *md)
     params[0] = OSSL_PARAM_construct_ulong(OSSL_CIPHER_PARAM_FLAGS, &v);
     ok = evp_do_md_getparams(md, params);
 
-    return ok != 0 ? v : 0;
+    return ok != 0 ? (int)v : 0;
```

Comment:
This is an unnecessary cast

Reply:
it will give a warning on windows - does that make is unnecessary?

Reply:
Fixed